doc/include/api_stubs.inc

```diff
@@ -129,45 +103,32 @@ current [file .critcl] file.
 [call [cmd ::critcl::api] [method extheader] [opt [arg file]...]]
 
-This method is similar [cmd {::critcl::api header}], in that it
-notifies critcl of companion header files which have to be exported
-together with the generated stubs headers.
-
-[para] The difference is that these headers will be expected to exist
-in the external development environment.
-
-As such they will be [term {#include}]d in the generated header for
-the package, but not copied to the package header directory. Nor are
-they allowed to be glob patterns, as critcl has no context, i.e
-directory, in which to expand such patterns.
+Like [cmd {::critcl::api header}], but each [arg file] should exist in the
+external development environment.  An [term {#include}] directive is added to
+[file [var foo]Decls.h], but [arg fiile] is not copied to the package header
+directory. [arg file] is not a glob pattern as Critcl has no context,
+i.e directory, in which to expand such patterns.
 
 [list_end]
 
-Note that the generated headers for an exported API are included in
-the package like it is done when importing it somewhere else. To repeat:
-
-[para] The [file [var foo]Decls.h] header is included at the beginning
-of the C code backing the [file .critcl] file, and at the beginning of
-all companion [file .c] files. This means that the writer of these
-files doesn't have to write the necessary [term {#include}] directory,
-critcl does it for them.
-
-[para] In mode "compile & run" the generated header files, and their
-companion headers, if any, are placed in the subdirectory [file foo]
-of the [sectref {Result Cache}]. As this location is implicitly added
-to the include search path any other package importing this API and
-and build in mode "compile & run" as well will find the these headers.
-
-[para] For mode "generate package" the application was extended with a
-new option [option -includedir] which specifies the location to place
-the generated headers in (again in subdirectory [file foo] of that
-path). This path is also be added to the include search paths,
-ensuring that a package importing an API will find it if the package
-exporting that API used the same setting for [option -includedir].
-
-[para] For mode "generate TEA" the static scanner was extended to
-recognize [cmd {critcl::api header}] as a source of companion files.
-It further uses data about [cmd {critcl::api import}] commands to put
-proper support for [option --with-[var foo]-include] options into the
-generate [file configure(.in)] so that a user may specify custom
+As with the headers for an imported API, an [term {#include}] directive is
+added to the generated code for the [vset critcl_script] and to
+each companion [file .c] file.
+
+[para] In "compile & run" mode the generated header files and any companion
+headers are placed in the [sectref {Result Cache}] subdirectory for the
+[vset critcl_script]. This directory is added to the include search path any
```

Comment:
`any` -> `of any`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).